specifyweb/frontend/js_src/lib/components/Toolbar/DataModel.tsx

```diff
@@ -297,11 +311,35 @@ function DataModelRelationships({
   readonly model: SpecifyModel;
 }): JSX.Element {
   const data = React.useMemo(() => getRelationships(model), [model]);
+
+  const [filteredDependentData, setFilteredDependentData] =
+    React.useState(data);
+  const [dependentFilter, setDependentFilter] = React.useState(false);
+
+  React.useEffect(() => {
+    if (dependentFilter === true) {
+      const dependentData = data
+        .filter((table) => table.isDependent === 'Yes')
```

Comment:
1. `table` is not a good varialbe names as tables can't be dependent - relationships can
2. This will likely not work when users change their interface language. Either modify `getRelationships` to have a two-item array for the `isDependent` option (1st item for sorting & filtering, 2nd for display), or get the relationship object from the relationship name: `model.strictGetRelationship(relationship.name[0]).isDependent`